ui/StatusQ/src/StatusQ/Core/LocaleUtils.qml

```diff
@@ -38,6 +38,20 @@ QtObject {
         }
     }
 
+    function getLocalizedDigitsCount(str, locale = null) {
+        if (!str)
+            return 0
+
+        locale = locale || Qt.locale()
+
+        const localizedNumbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9].map(
+                n => LocaleUtils.numberToLocaleString(n, 0, locale))
+
+        const regExp = new RegExp(`[^${localizedNumbers.join("")}]`, "g")
```

Comment:
These 2 `const`s could actually be properties inside `d`, in order not to get evaluated each time this is called (on each keypress in StatusInput)

Reply:
it's quite problematic here to do it in a generic way bc it depends on locale

Reply:
ok, I did some caching here, not the cleanest solution, but correct from LocaleUtils' api perspective. Let me know if you have any better idea for that :)